weaviate/collections/classes/tenants.py

```diff
@@ -62,11 +62,47 @@ class TenantActivityStatusInput(str, Enum):
 
     HOT = "HOT"
     COLD = "COLD"
+
+
+class TenantUpdateActivityStatus(str, Enum):
+    """TenantActivityStatus class used to describe the activity status of a tenant to update in Weaviate.
+
+    Attributes:
+        `HOT`
+            The tenant is fully active and can be used.
+        `COLD`
+            The tenant is not active, files stored locally.
+    """
+
+    HOT = "HOT"
+    COLD = "COLD"
     FROZEN = "FROZEN"
```

Comment:
frozen is missing from docstring - are we expecting that create and update will have different activity statuses i the future?

Reply:
Yes, this is the current backend behaviour. It is not possible to create a tenant in `FROZEN` state, only `HOT` and `COLD`. You can only update a tenant to `FROZEN` state. Since we have this distinction now, I feel it best to communicate this in the client API with it likely that other differences in behaviour may arise in the future. Having a split is the easiest way to avoid annoying deprecation cycles if we need to split them later, like we're having to do now with `Tenant -> TenantCreate/TenantUpdate` due to the read-only `FREEZING/UNFREEZING/UNFROZEN` states

Reply:
makes sense, but frozen should still be added to the docstring :)

Reply:
Yeh, just realised after I made the comment. Have committed :D